lib/boleto/boleto.ex

```diff
@@ -42,142 +41,185 @@ defmodule StarkBank.Boleto do
     - status [string, default nil]: current Boleto status. ex: "registered" or "paid"
     - created [DateTime, default nil]: creation datetime for the Boleto. ex: ~U[2020-03-26 19:32:35.418698Z]
   """
-  @enforce_keys [:amount, :name, :tax_id, :street_line_1, :street_line_2, :district, :city, :state_code, :zip_code]
-  defstruct [:amount, :name, :tax_id, :street_line_1, :street_line_2, :district, :city, :state_code, :zip_code, :due, :fine, :interest, :overdue_limit, :tags, :descriptions, :id, :fee, :line, :bar_code, :status, :created]
+  @enforce_keys [
+    :amount,
+    :name,
+    :tax_id,
+    :street_line_1,
+    :street_line_2,
+    :district,
+    :city,
+    :state_code,
+    :zip_code
+  ]
+  defstruct [
+    :amount,
+    :name,
+    :tax_id,
+    :street_line_1,
+    :street_line_2,
+    :district,
+    :city,
+    :state_code,
+    :zip_code,
+    :due,
+    :fine,
+    :interest,
+    :overdue_limit,
+    :tags,
+    :descriptions,
+    :id,
+    :fee,
+    :line,
+    :bar_code,
+    :status,
+    :created
+  ]
 
   @type t() :: %__MODULE__{}
 
   @doc """
   Send a list of Boleto structs for creation in the Stark Bank API
 
   ## Parameters (required):
-    - user [Project]: Project struct returned from StarkBank.project().
     - boletos [list of Boleto structs]: list of Boleto structs to be created in the API
 
+  ## Keyword Args:
+    - user [Project] (optional): Project struct returned from StarkBank.project().
+
   ## Return:
     - list of Boleto structs with updated attributes
   """
   @spec create(Project.t(), [Boleto.t()]) ::
```

Comment:
I noticed that the @spec of the following functions seems to be incorrect for all resources: create, create!, get, get!, pdf, pdf!, delete, delete!

Since now the user can be passed only as an option, shouldn't it be on the following format instead?

`@spec create([Boleto.t()], any) :: {:ok, [Boleto.t()]} | {:error, [Error.t()]}`

Reply:
Just pushed a fix